test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,23 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(500);
+const N = 5;
 var nbIntervalFired = 0;
-var timer = setInterval(function() {
+
+const keepOpen = setTimeout(function () {
+  assert.strictEqual(nbIntervalFired, N);
+}, TEST_DURATION);
+
+const timer = setInterval(function() {
```

Comment:
Could you run `make jslint`? It should pick up the difference between the spacing around the parenthesis here vs above. If you want, you could use arrow functions like `() => {` although that's definitely not necessary.